components/experimental/src/duration/validated_options.rs

```diff
@@ -187,6 +187,14 @@ impl ValidatedDurationFormatterOptions {
             prev_style = *style;
         }
 
+        // section 1.2.1
+        // 27. Set durationFormat.[[FractionalDigits]] to ? GetNumberOption(options, "fractionalDigits", 0, 9, undefined).
```

Comment:
GetNumberOption throws a RangeError if it is out of range.

https://tc39.es/ecma402/#sec-defaultnumberoption

Reply:
Thanks! You're right, I misread...

Fixed now.